### DISCUSSION TITLE
Update to VS 17.5

### DISCUSSION THREAD

**Discussion by pmiossec** [gitextensions/gitextensions#10859] [General]
I don't know how Microsoft did but when you open the last version v17.5 after update with a project, for example "GitExtensions", it give you the performance improvements you can expect.... I see no other good explanation on why GitExtensions build times are displayed in the feature description 🤣🤣🤣🤣🤣🤣 

![image](https://user-images.githubusercontent.com/460196/229937333-228afa9b-0a83-4cef-ac59-e8e5845b54dd.png)

**Comment by RussKie** [comment]
That's probably because @drewnoakes did that 🤣
